File: tests/integration_tests/tutorial_fr/programs/tokens_fr.py
# jetons 1 to 5

def ramasse_tout():
    while objet_ici():
        prend()
        avance()

def deposte_tout_et_avance():
    while transporte():
        dépose()
    avance()
    
while not au_but():
    if objet_ici():
        ramasse_tout()
    elif transporte():
        deposte_tout_et_avance()
    else:
        avance()

File: tests/integration_tests/tutorial_fr/programs/tokens_fr.js
// tokens 1 to 5

function ramasse_tout() { 
    while (objet_ici()) { 
        prend (); 
        avance (); 
    }
}

function depose_tout_et_avance () { 
    while (transporte()) { 
        depose (); 
    }
    avance (); 
}
    
while (!at_goal()) { 
    if (objet_ici()) { 
        ramasse_tout(); 
    }
    else if (transporte()) { 
        depose_tout_et_avance (); 
    }
    else {
        avance (); 
    }
}
